inference-engine/src/mkldnn_plugin/nodes/detectionoutput.cpp

```diff
@@ -263,8 +263,11 @@ class DetectionOutputImpl: public ExtLayerBase {
                     }
                 }
 
-                std::sort(conf_index_class_map.begin(), conf_index_class_map.end(),
-                          SortScorePairDescend<std::pair<int, int>>);
+                std::sort(conf_index_class_map.begin(),
+                          conf_index_class_map.end(),
+                          [](const std::pair<float, std::pair<int, int>>& p1, const std::pair<float, std::pair<int, int>>& p2) {
+                              return (p1.first > p2.first) || (p1.first == p2.first && p1.second.second < p2.second.second);
+                          });
```

Comment:
why not using SortScorePairDescend ? it is already fixed aligned to this

Reply:
fixed